### PR TITLE
Change itermitems to Support Python 3

### PR DESCRIPTION
**Pull request by shblhy**
* change dict.iteritems() to six.iteritems(dict)
* change _to_string return str not bytes

Signed-off-by: Yan Huang <345471536@qq.com>

### CODE REVIEW

File: jaeger_client/thrift.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ def id_to_int(big_id):
 
 def _to_string(s):
     try:
+        if isinstance(s, str):
+            return s
```
Comment (yurishkuro) [inline jaeger_client/thrift.py]: this will skip encoding in py3

File: jaeger_client/thrift_gen/agent/Agent.py [hunk 2]
```diff
@@ -209,7 +209,7 @@ def __hash__(self):
 
   def __repr__(self):
     L = ['%s=%r' % (key, value)
-      for key, value in self.__dict__.iteritems()]
+      for key, value in six.iteritems(self.__dict__)]
```
Comment (yurishkuro) [inline jaeger_client/thrift_gen/agent/Agent.py]: this makes sense, but we don't edit generated files manually, there is a section in the makefile that rewrites some of the code.

Comment (shblhy) [reply jaeger_client/thrift_gen/agent/Agent.py]: I think you use thrift and .thirft file to gen ttypes.py file,  where can i get the .thirft files?

Comment (yurishkuro) [reply jaeger_client/thrift_gen/agent/Agent.py]: `.thrift` files are in `jaeger-idl`, run `git submodule update --init`

Comment (yurishkuro) [reply jaeger_client/thrift_gen/agent/Agent.py]: actually `make thrift` will do everything.

Comment (yurishkuro) [reply jaeger_client/thrift_gen/agent/Agent.py]: see https://github.com/jaegertracing/jaeger-client-python/blob/master/CONTRIBUTING.md#thrift

**Comment by yurishkuro** [comment]
Py3 compatibility has been achieved with other PRs

**Pull request closed without merging**